frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -17,14 +17,16 @@ export const scene: SceneExport = {
 
 const WARNING_TYPE_TO_DESCRIPTION = {
     cannot_merge_already_identified: 'Refused to merge an already identified user via $identify or $create_alias call',
+    cannot_merge_with_illegal_distinct_id:
+        'Refused to merge with an illegal distinct_id via $identify or $create_alias call',
+    cannot_merge_attepts_exhausted: 'Merge via $identify or $create_alias call failed',
+    skipping_event_invalid_uuid: 'Refused to process event with invalid uuid',
```

Comment:
This is missing a link to documentation on how users might trigger this issue and how to fix this.